examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -429,3 +447,46 @@ export const TaskListInstantiationFactory = new DataObjectFactory<TaskList>(
 	[SharedCell.getFactory(), SharedString.getFactory(), SharedMap.getFactory()],
 	{},
 );
+
+export class BaseDocument extends DataObject implements IBaseDocument {
```

Comment:
Nit: consider moving this and its instantiation factory to a separate file. Doesn't need to happen in this PR.